### ISSUE TITLE
Sidebar Domnload count unreadable

### ISSUE THREAD

**Issue by LuvBiglyBT** [BiglySoftware/BiglyBT#367]
I remeber I encountered this with Vuze and found a way to change it. I have gone into every option that I think might address the problem, but why is the only count that is green Downloads?  There is not enough contrast between the black number and the green background, at least for me.  It just looks green to me.  I personally like to check on how many torrents are still downloading without opening the Download page completely.  Can we either change it to the same colow as the other count windows in the sidebar or give us the option to change it to to colors that have more contrast?  Maybe I am missing something, but probably not. :)

**Comment by parg** [comment]
I'll add an option to leave all sidebar indicators the default colour - for me the green/black is legible but obviously this isn't universal! Thanks

**Comment by parg** [comment]
its is a right-click option on the Downloading sidebar entry - please check the next beta when built (B20)

**Comment by LuvBiglyBT** [comment]
Thanks Parg.  Just a quick question: I get notifications of Beta updates but the updater does not pop up.  When I do a manual check for updates, I get the message that I already have the newest stable beta version, and it ignores the new one that  I receive notification for.  Is this normal?  Does it take time to funnel through the system and become available for download?  Also, is anonymous update supposed to be checked or unchecked?  I thought I finally got B17 by unchecking it, then I got B18 only after checking it.  :)

**Comment by parg** [comment]
I know some people have issues with 'anonymous updates' checked but I've never heard of the opposite! BiglyBT does cache the latest version state retrieved from the version server for some time so it is possible that you might see a new beta version advertised on the blog but a 'check for updates' say nothing was available. Not sure of the cache period off the top of my head

**Comment by LuvBiglyBT** [comment]
Hey there Parg!

 

OK, I give up, where is the option to change the color for the Downloading view?  I have searched and searched and cannot find it in options :).

 

One other observation, the check for updates is still spotty.  I am still doing my own troubleshooting to figure out if it is being blocked by my security or not.  Checking and unchecking anonymous seems very finicky.  I am not absolutely sure yet, but it seems to go back and forth in its effect. I will keep you posted.

 

Regards,

LuvBiglyBT

**Comment by LuvBiglyBT** [comment]
LOL…never mind on the option front.  I found it. :)

 

 

Hey there Parg!

 

OK, I give up, where is the option to change the color for the Downloading view?  I have searched and searched and cannot find it in options :).

 

One other observation, the check for updates is still spotty.  I am still doing my own troubleshooting to figure out if it is being blocked by my security or not.  Checking and unchecking anonymous seems very finicky.  I am not absolutely sure yet, but it seems to go back and forth in its effect. I will keep you posted.

 

Regards,

LuvBiglyBT

**Comment by parg** [comment]
one down!

**Comment by LuvBiglyBT** [comment]
Parg,

 

I have done enough testing on the update to be able to report back to you.  Sometimes I have to check anonymous and sometimes I have to uncheck it to get the updates.  I wish I could give you more specific reasons, but the best I can figure is that it just needs a change, as odd as that sounds.  Where ever it is , it needs the opposite the next time I want to download.

 

This morning I had a different experience.  No matter how it was checked, it would not download until I checked PeerBlock to see if an IP was being blocked.  I was concerned when I saw that an IPs that are on a US Government Blacklist were being blocked.  There were two: 34.203.76.220 and 34.224.200.142.  Once I white listed them temporarily, the update was downloaded.  PeerBlock for me is secondary to other security that I have, so I am not that concerned, but I think it may be worth noting.

 

 

Best,

LuvBiglyBT

 

 

LOL…never mind on the option front.  I found it. :)

 

 

Hey there Parg!

 

OK, I give up, where is the option to change the color for the Downloading view?  I have searched and searched and cannot find it in options :).

 

One other observation, the check for updates is still spotty.  I am still doing my own troubleshooting to figure out if it is being blocked by my security or not.  Checking and unchecking anonymous seems very finicky.  I am not absolutely sure yet, but it seems to go back and forth in its effect. I will keep you posted.

 

Regards,

LuvBiglyBT
